## Search literature relevant to the following questions:
 - relate the distances (phylogenetic or % difference) of nuclear genomes with taxonomic units
 - How far to another sequenced genome is a given genome?
   - A few articles trying to classify Bacteria using Genome Similarity-Based Taxonomy:  
     Boris A. Vinatzer, Alexandra J. Weisberg, Caroline L. Monteil, Haitham A. Elmarakeby, Samuel K. Sheppard, and Lenwood S. Heath (2016)  
     Adeolu M, Alnajar S, Naushad S, S. Gupta R. (2016)
   - And discussing Microbial Taxonomy based of genome comparison  
     Thompson, C.C., Amaral, G.R., Campeão, M. et al.
 - Distances in regions of nuclear genome that is not functional, e.g. introns of orthologous genes.
 - Definition of taxonomic units (e.g. family, order). Any relationship to genetic distance?
   - Oh C, Lee K, Cheong Y, Lee SW, Park SY, et al. (2015). Research of oral microbiomes of canines. For the taxomic assignment of the sequence reads they used NCBI Taxonomy NCBI. However, for the taxomiy based on similarity, they use some "Taxonomical guidelines": Species >97%, Genus >94%, Family >90%, Order >85%, Class >80%, Phylum >75%.  
	  They cite a paper after naming these "Taxonomical Guidelines" but I did find nothing on the cited paper.
 - Papers that use NCBI taxonomy in a systematic or automatic way.  
   - Podell S, Gaasterland T, Allen E (2008). Developed an algorithm to better detect HGT. The program execution requires locally available copies of the NCBI Genbank database and also NCBI Taxonomy.

Tips: http://scholar.google.com, https://www.ncbi.nlm.nih.gov/pubmed/

## Papers
  - “The NCBI Taxonomy database”, Scott Federhen, Nucleic Acids Research, Volume 40, Issue D1, 1 January 2012, Pages D136-D143, https://doi.org/10.1093/nar/gkr1178, https://academic.oup.com/nar/article/40/D1/D136/2903327,  
__highly relevant__
  - “Assembly: a resource for assembled genomes at NCBI”, Paul A. Kitts et al., Nucleic Acids Research, Volume 44, Issue D1, 4 January 2016, Pages D73-D80, https://academic.oup.com/nar/article/44/D1/D73/2502704
  - “A perspective on 16S rRNA operational taxonomic unit clustering using sequence similarity”, Nam-Phuong Nguyen, Tandy Warnow, Mihai Pop and Bryan White npj Biofilms and Microbiomes, volume 2, Article number: 16004 (2016), https://www.nature.com/articles/npjbiofilms20164
 - "Type material in the NCBI Taxonomy Database", Scott Federhen, Nucleic Acids Research, Volume 43, Database Issue, 14 November 2014, Pages D1086-D1098, https://doi.org/10.1093/nar/gku1127  
   _Similar to the "The NCBI Taxonomy Paper" explaining how the type material works in the dabase._
 - Oh C, Lee K, Cheong Y, Lee SW, Park SY, et al. (2015) Comparison of the Oral Microbiomes of Canines and Their Owners Using Next-Generation Sequencing. PLOS ONE 10(7): e0131468. https://doi.org/10.1371/journal.pone.0131468  
   _taxonomic units based on similarity_
 - Boris A. Vinatzer, Alexandra J. Weisberg, Caroline L. Monteil, Haitham A. Elmarakeby, Samuel K. Sheppard, and Lenwood S. Heath (2016) A Proposal for a Genome Similarity-Based Taxonomy for Plant-Pathogenic Bacteria that Is Sufficiently Precise to Reflect Phylogeny, Host Range, and Outbreak Affiliation Applied to Pseudomonas syringae sensu lato as a Proof of Concept, Phytopathology 2017 107:1, 18-28  
   _Genome Similarity-Based Taxonomy, Genome comparison_
 - Adeolu M, Alnajar S, Naushad S, S. Gupta R. (2016), Genome-based phylogeny and taxonomy of the ‘Enterobacteriales’: proposal for Enterobacterales ord. nov. divided into the families Enterobacteriaceae, Erwiniaceae fam. nov., Pectobacteriaceae fam. nov., Yersiniaceae fam. nov., Hafniaceae fam. nov., Morganellaceae fam. nov., and Budviciaceae fam. nov., Int J Syst Evol Microbiol 66(12):5575-5599 doi:10.1099/ijsem.0.001485  
   _Genome Similarity-Based Taxonomy, Genome comparison_
 - Thompson, C.C., Amaral, G.R., Campeão, M. et al. Arch Microbiol (2015) 197: 359. https://doi.org/10.1007/s00203-014-1071-2  
   _Microbia Taxonomy Discussion_
 - Podell S, Gaasterland T, Allen E (2008) A database of phylogenetically atypical genes in archaeal and bacterial genomes, identified using the DarkHorse algorithm. BMC Bioinformatics **9**:419, https://doi.org/10.1186/1471-2105-9-419  
   _Using NCBI Taxonomy_
